requirements.txt

```diff
@@ -0,0 +1,2 @@
+networkx==2.8.5
+numpy==1.22.4
```

Comment:
`networkx` depends on `numpy>=1.20`, so we might be able to just specify the former unless there is a specific reason for `numpy==1.22.4`.